docs/user-guides/1-metachromium-overview.md

```diff
@@ -3,32 +3,26 @@ id: metachromium-overview
 title: Metachromium Overview
 ---
 
-[Metachromium](https://metachromium.com/) is a spatial app browser that runs side-by side with OpenVR (SteamVR). It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
+<a href="https://metachromium.com/" target="_blank" rel="noopener noreferrer">Metachromium</a> is a spatial app browser that runs side-by side with OpenVR (SteamVR). It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
 
 ## Features
 
 - Run WebXR context on top of an OpenVR application
-
 - Run multiple WebXR applications inside a single Metachromium context
-
 - Volumetric streaming capabilities
 
 ## Using Metachromium
```

Comment:
I'm actually not sure if Metachromium should be part of these docs at all. It is a mostly unrelated project with similar ideas, but it is not required for anything we're doing here.

When packages become default UI for Metachromium then maybe we could add it back.

Reply:
I would apply that to MetaRTC as well -- it is an open source library we offer and isn't necessarily related to the rest of the effort. It might make more sense to mention it in multiplayer examples.

Reply:
I see, that makes sense.

We could condense and add a mini paragraph on each in another 'Other works' or similar section maybe?

Reply:
Do you think there's anything else we could add to replace those two sections 'for users'? Or if we should even have anything 'for users' at all!

Reply:
> We could condense and add a mini paragraph on each in another 'Other works' or similar section maybe?

Yeah, maybe "other Webaverse projects".

Reply:
Resolved in https://github.com/webaverse/docs/pull/17